static/vendor/data/amp-iframe.html

```diff
@@ -0,0 +1,14 @@
+<html>
```

Comment:
Maybe add a link to the documentation file you created with more info about this file? Just so it's easier to find.

Reply:
The documentation file lives in the same folder as I didn't want to ship comments to a documentation to our users :)